demo/docs/features/errors.md

```diff
@@ -0,0 +1,53 @@
+# Errors and Exceptions
+
+All apps will eventually throw an error that will be treated by developers or exceptions when something unforeseen
+happens.
+
+## Description
+
+### Client
+
+In the case of errors (i.e. HTTP calls that return `4xx` or `5xx` status codes), we wrap the call in a `try/catch` block
+for `async/await` operations or using `then` and `catch` for promises and use the `pushError` API provided by Grafana
+JavaScript Agent to report them.
+
+Exceptions are reported automatically as Grafana JavaScript Agent offers a way to automatically capture exceptions.
+
+**The captured errors are stored in Loki.**
+
+### API
+
+The errors and exceptions that are captured server side are reported manually by writing them to a file which is then
+read by the [Grafana Agent](https://github.com/grafana/agent).
+
+**The captured errors are stored in Loki.**
+
+**Given the way how the API reports the errors, they can't be filtered by `kind=exception` since they will appear as
```

Comment:
To clarify, can't or can be filtered by kind? Might've missed a discussion around this so feel free to ignore, but seems like we should include kind=exception on error events so they're easy to filter?